Class.js

```diff
@@ -4,6 +4,9 @@ class Class {
     this.teacher = teacher
     this.students = []
   }
-}
 
-module.exports = Class;
+  addStudent = student => {
```

Comment:
there is no arrow functions in classes. You would have to write it like `addStudent( student ){ your code here }`

Reply:
This needs to be changed for the file to run